frontend/pages/ManageControlsPage/OSSettings/cards/DiskEncryption/components/DiskEncryptionTable/DiskEncryptionTableConfig.tsx

```diff
@@ -108,21 +108,32 @@ const defaultTableHeaders: IDataColumn[] = [
     ),
     disableSortBy: true,
     accessor: "windowsHosts",
-    Cell: ({
-      cell: { value: aggregateCount },
-      row: { original },
-    }: ICellProps) => {
+    Cell: ({ cell: { value: aggregateCount } }: ICellProps) => {
       return (
-        <div className="disk-encryption-table__aggregate-table-data">
-          <TextCell value={aggregateCount} formatter={(val) => <>{val}</>} />
-          <ViewAllHostsLink
-            className="view-hosts-link"
-            queryParams={{
-              [HOSTS_QUERY_PARAMS.DISK_ENCRYPTION]: original.status.value,
-              team_id: original.teamId,
-            }}
-          />
-        </div>
+        <TextCell value={aggregateCount} formatter={(val) => <>{val}</>} />
+      );
+    },
+  },
+  {
+    title: "",
+    Header: "",
+    accessor: "linkToFilteredHosts",
```

Comment:
Refactored all "View all hosts" links to follow the same pattern (Having a separate column)